lib/index.js

```diff
@@ -150,6 +154,25 @@ module.exports = class OtpRunner {
     )
   }
 
+  /**
+   * Returns true if OTP 2.x is being used.
+   */
+  isUsingOtp2x () {
+    return this.manifest.otpVersion === '2.x'
+  }
+
+  /**
+   * Returns the appropriate folder to use for downloading files to and for
+   * where the Graph.obj file will be written to.
```

Comment:
Minor tweak:

> Returns the appropriate folder to use for writing both downloaded files and the Graph.obj file.

Reply:
Refactored in 2610ee9df6cb1878f067a5e2c1c277eed33836fe